packages/types/src/integration.ts

```diff
@@ -39,6 +39,12 @@ export interface IntegrationFnResult {
    */
   setup?(client: Client): void;
 
+  /**
+   * This hook is triggered after `setupOnce()` and `setup()` have been called for all integrations.
+   * You can use it if it is important that all other integrations have been run before.
+   */
+  afterSetup?(client: Client): void;
```

Comment:
l: wdyt about `afterIntegrationsSetup` or `afterAllSetup`? To me `afterSetup` conveys more that it'd be called once the integration implementing it was set up. 
Don't wanna 🚲 shed to long though, I'm also fine if we leave it as is.